hack/run-ci-e2e-test.sh

```diff
@@ -101,6 +101,27 @@ if ! [[ "$OPENSHIFT_CI" = "true" &&  "$TEST" = "upgrade" ]]; then
   done
 fi
 
+# WINDOWS_INSTANCES_DATA holds the windows-instances ConfigMap data section
+WINDOWS_INSTANCES_DATA=${WINDOWS_INSTANCES_DATA:-}
+# Check windows-instances data and create the windows-instances ConfigMap, if
+# any. The ConfigMap creation must takes place after the operator's deployment
+# process is complete, because the namespace is removed as part of the cleanup
+# step (cleanup_WMCO) while retrying the deployment.
+if [[ -n "$WINDOWS_INSTANCES_DATA" ]]; then
+  # create ConfigMap
+  cat <<EOF | oc apply -f -
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: windows-instances
+  namespace: ${WMCO_DEPLOY_NAMESPACE}
+${WINDOWS_INSTANCES_DATA}
+EOF
+  # update BYOH node count from ConfigMap
+  BYOH_NODE_COUNT_OPTION="--byoh-node-count=$(oc get configmaps windows-instances  -n $WMCO_DEPLOY_NAMESPACE -o json | jq '.data | length')"
+  echo "updated ${BYOH_NODE_COUNT_OPTION}"
+fi
+
```

Comment:
How about breaking this out into a bash function?

Reply:
I can see the ConfigMap creation (lines 111 to 119) modularized in a separated function. But, the `WINDOWS_INSTANCES_DATA` handling is part of this execution flow.

Reply:
`WINDOWS_INSTANCES_DATA` can be passed to the bash function

Reply:
Introduced 
```
createWindowsInstancesConfigMap "${WINDOWS_INSTANCES_DATA}"
```